app/javascript/views/Home.vue

```diff
@@ -0,0 +1,60 @@
+<template>
+  <div id="app" class="bg-gray-300">
+    <div id="app__body" class="flex">
+      <div
+        id="app__bodyLeft"
+        class="bg-blue-400 p-5 h-screen w-1/4 overflow-scroll"
+      >
+        <div class="grid grid-cols-2 place-items-end">
+          <div class="text-white font-bold">Hay {{ numberOfGroups }} grupos:</div>
+          <div>
+            <router-link class="text-end font-bold text-xl text-blue-400 bg-white rounded-full h-8 w-8 flex items-center justify-center" to="/addTeam">+</router-link>
```

Comment:
Esta linea está muy larga (aunque es principalmente culpa de la clase css), puedes mover cosas a otras lineas
```suggestion
            <router-link
              class="text-end font-bold text-xl text-blue-400 bg-white rounded-full h-8 w-8
                flex items-center justify-center"
              to="/addTeam"
            >
              +
            </router-link>
```